docs/articles/new-expensify/expenses-&-payments/Send-an-invoice.md

```diff
@@ -90,5 +90,4 @@ When invoicing is disabled, all previously created invoice rooms and historical
 
 We can only accept a private domain website to ensure the security of your business. If you receive an error when entering your website, it is likely because the domain is not recognized as private. Make sure you are using a business email with a private domain. If you continue to experience issues, contact our support team at concierge@expensify.com for further assistance.
 
-
-</div>
+{% include faq-end.md %}
```

Comment:
We don't want to delete the `</div>`. We have an opening `<div>` tag on line 5 of this file, and this is its closing tag. Please [see this article](https://github.com/Expensify/App/blob/main/docs/articles/expensify-classic/bank-accounts-and-payments/bank-accounts/Connect-Personal-US-Bank-Account.md?plain=1) for an example of what we want. 

It should be like this:
```
{% include faq-end.md %}

</div>
```

Reply:
Ah, okay, I updated the article!